index.html

```diff
@@ -10,26 +10,41 @@
 <body>
     <nav class="top-nav">
         <div class="left-nav fs-1">
-            <a href="#" class="fs-1">Awesome Books</a>
+            <a href="#" class="fs-1" id="AwesomeBooksProject">Awesome Books</a>
         </div>
         <div class="right-nav">
-            <a href="#" class="fs-1">List</a>
-            <a href="#" class="fs-1">Add new</a>
-            <a href="#" class="fs-1">Contact</a>
+            <a href="#" id="BookList" class="fs-1">BOOK LIST</a>
+            <a href="#" id="AddBook" class="fs-1">ADD BOOK</a>
+            <a href="#" id="Contact-us" class="fs-1">CONTACT US</a>
         </div>
     </nav>
```

Comment:
- [ ] I think you have missed to add the the current date that should displayed below the navigation bar, for this requirement, you **should** use the [Luxon](https://moment.github.io/luxon/docs/manual/install.html) library. See [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time). Please fix that.

As you can see in this picture, below the navbar current date and time is showing.

### wireframe design

![Screenshot from 2021-07-08 19-18-12](https://user-images.githubusercontent.com/22434660/124933545-9a2b3e80-e021-11eb-88d9-58eedad1f968.png)